renderdoc/driver/vulkan/wrappers/vk_cmd_funcs.cpp

```diff
@@ -7609,16 +7614,22 @@ bool WrappedVulkan::Serialise_vkCmdBuildAccelerationStructuresIndirectKHR(
       tempmemSize += GetNextPatchSize(&pInfos[i]);
 
     byte *memory = GetTempMemory(tempmemSize);
-    auto *unwrappedInfos = reinterpret_cast<VkAccelerationStructureBuildGeometryInfoKHR *>(memory);
+    VkAccelerationStructureBuildGeometryInfoKHR *unwrappedInfos =
+        (VkAccelerationStructureBuildGeometryInfoKHR *)memory;
     memory += sizeof(VkAccelerationStructureBuildGeometryInfoKHR) * infoCount;
 
     for(uint32_t i = 0; i < infoCount; ++i)
       unwrappedInfos[i] = *UnwrapStructAndChain(m_State, memory, &pInfos[i]);
 
+    // Convert the maxPrimitives back to a C-style array-of-arrays
+    rdcarray<const uint32_t *> tmpMaxPrimitiveCounts(nullptr, infoCount);
```

Comment:
Please use a normal `NULL` rather than `nullptr` here.

Reply:
Will do.  Out of curiosity why aren't C++ features used?  I know `auto` is pretty contentious in the community but why the aversion to `nullptr` and C++ cast operators?

Reply:
I don't see any value in them at all, so in my view they make the code less readable for no gain.